README.md

```diff
@@ -164,3 +180,21 @@ device, and then copy the modules on to your device.
 ## Running as a service
 
 An example service file is provided [here](https://github.com/flowforge/flowforge-device-agent/tree/main/service).
+
+## Running the agent with the Web UI enabled
+
+The agent can be run with a Web UI interface. This is an optional feature and
+is not enabled by default.  The UI permits the user to view the current status of 
+the agent and the Node-RED instances it is running.  It also permits
+the user to download a device configuration or a provisioning configuration file.
```

Comment:
```suggestion
The agent can be run with a Web UI interface. This is an optional feature and
is not enabled by default.  The UI permits the user to view the current status of 
the agent and the Node-RED instances it is running.  It also permits
the user to download a device configuration or a provisioning configuration file.
```

I think the emphasis needs to be about this being a UI for doing initial configuration/setup of the agent - it isn't a general purpose Web UI for monitoring the status of the agent, especially given it shuts down after a short period of time.